cnames_active.js

```diff
@@ -1916,6 +1916,7 @@ var cnames_active = {
   "mzplayer": "prince3661.github.io/Mzplayer",
   "n": "cyb.github.io/n",
   "nabin": "nabin6246.github.io",
+  "nacl-check": "magnusmarx.github.io/NaCL/",
```

Comment:
```suggestion
  "nacl-check": "magnusmarx.github.io/NaCL",
```
Per the CI error, you don't need a trailing slash